doc/cephfs/nfs.rst

```diff
@@ -79,3 +79,222 @@ Current limitations
 
 - Per running ganesha daemon, FSAL_CEPH can only export one Ceph file system
   although multiple directories in a Ceph file system may be exported.
+
+Exporting over NFS clusters deployed using rook
+===============================================
+
+In this tutorial we will be using minikube for setting up a single node test
+cluster. If you already have a kubernetes cluster deployed, please directly
+start with rook part.
+
+.. note:: Configuration of this tutorial should not be used in a a real
+          production cluster. For the purpose of simplification, the security
+          aspects of Ceph are overlooked in this setup.
+
+Minikube Setup
+--------------
+
+This setup is done on Fedora 30 and following package versions were used::
+
+        minikube v1.4.0, Kubernetes v1.16.0, Docker 18.09.9, git 2.21.0, kubernetes-client 1.15.2-1, docker-common 1.13.1-68 and qemu-kvm 3.1.1-2.
+
+Installation::
+
+        dnf install kubernetes-client docker-common git qemu-kvm
+        cd /tmp
+        curl -Lo minikube https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64 && chmod +x minikube
+        curl -LO https://storage.googleapis.com/minikube/releases/latest/docker-machine-driver-kvm2 && chmod +x docker-machine-driver-kvm2
+        cp minikube docker-machine-driver-kvm2 /usr/local/bin
+
+.. note:: Place the minkube and docker executables in $PATH. Make sure $HOME has enough space as minikube requires about 25 GB.
+
+
+Deploy Minikube Cluster::
+
+        minikube start --vm-driver=kvm2
+
+.. note:: Add --memory and --cpus options to provide custom RAM and CPU values for the VM.
+
+List the running pods::
+
+        kubectl get pods --all-namespaces
+
+Rook Setup And Cluster Deployment
+---------------------------------
+
```

Comment:
I suggest adding a reference link somewhere in this section,
https://rook.io/docs/rook/master/ceph-quickstart.html

Reply:
Done!